EKF/ekf_helper.cpp

```diff
@@ -1244,6 +1247,15 @@ void Ekf::updateBaroHgtOffset()
 	}
 }
 
+float Ekf::getGpsAltVar()
+{
+	// observation variance - receiver defined and parameter limited
+	const float lower_limit = fmaxf(_params.gps_pos_noise, 0.01f);
```

Comment:
Due to difference in GPS vertical vs horizontal error characteristics, I think this should be

const float lower_limit = fmaxf(1.5f*_params.gps_pos_noise, 0.01f);

Will set up a call to discuss.

Reply:
Yes, that makes sense